cr_module/classes/redfish.py

```diff
@@ -226,7 +227,51 @@ def restore_session_from_file(self):
         self.session_was_restored = True
 
         return
-
+    
+    def remove_session_lock(self):
+        if (self.cli_args.sessionlock is False or self.session_file_path is None):
+            return
+        try:
+            if os.path.exists(self.session_file_lock):
+                os.remove(self.session_file_lock)
+        except Exception as e:
+            self.exit_on_error(f"Unable to remove session lock '{self.session_file_lock}': {e}", "CRITICAL")
```

Comment:
Plugin internal errors should return UNKNOWN

Reply:
Done